content/2017/12/12/introducing-herman.rst

```diff
@@ -0,0 +1,389 @@
+public: yes
+tags: [Herman, CSS, Sass, 'Design Systems', Code, 'Style Guides']
+author: miriam
+image:
+  - src: 'blog/2017/herman-intro/header.jpg'
+headline:
+  - tagline: 'An agile approach to design systems'
+summary: |
+  **Design systems streamline development, communication, and consistency** --
+  but often rely on dedicated teams and extended budgets.
+  We wanted a tool to help us create and maintain
+  living style guides & pattern libraries
+  in an agile process, and on a budget.
+  `Herman`_ keeps our development flow simple,
+  and our UX consistent,
+  as we iterate on patterns
+  and scale over time.
+
+  .. _Herman: /herman/
+
+
+Herman Style Guides
+===================
+
+I spent the week after Thanksgiving in San Francisco,
+attending (and `accidentally speaking at`_)
+`Clarity`_ --
+a "`Design Systems`_" conference
+organized by the wonderful `Jina Anne`_.
+
+.. _accidentally speaking at: #@@@
+.. _Clarity: #@@@
+.. _Design Systems: #@@@
+.. _Jina Anne: #@@@
+
+I'm always curious how many attendees at a conference
+are working on enterprise projects
+over a long period of time,
+vs. those of us who do consulting
+for multiple clients each year.
+These different contexts can lead to very different solutions --
+affecting team size, budget, process, architecture,
+and technical requirements.
+
+I don't know the exact mix at Clarity,
+but I get the sense that design systems generally
+skew towards the enterprise projects for one reason: budget.
+Creating the beautiful `Salesforce Lightning Design System`_
+required a full-time dedicated team and budget.
+Those of us doing agency work don't often have that luxury.
+
+.. _Salesforce Lightning Design System: #@@@
+
+So how can we make design systems part of an agile process --
+growing slowly along-side the application,
+the same way we write and maintain test-coverage
+as part of any project?
+How do we make documentation and design consistency
+the path of least resistance?
+
+    If it's not documented, it doesn't exist.
+
+    --Miriam Suzanne (me)
+
+With `Herman`_ we can start small,
+documenting brand design tokens --
+`colors`_, `fonts`_, `sizes`_, `ratios`_, and `icons`_ --
+before expanding into `practical patterns and components`_,
+and with guides and systems that scale along with us.
```

Comment:
Again, can you make it clearer that this tool is not just for OddBird, but a free open-source tool for everyone in our position?

Reply:
Can we make that clear once at the top, and then take a personal approach to discussing how we use it?